<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectorWindow</class>
 <widget class="QMainWindow" name="DirectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="collectionListview"/>
      <widget class="QListView" name="groupListview"/>
      <widget class="QSplitter" name="sceneSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <layout class="QHBoxLayout" name="playerSelectionLayout"/>
       </widget>
       <widget class="EditorGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
       </widget>
       <widget class="QSplitter" name="drawableSplitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QListView" name="sceneListview"/>
        <widget class="QGroupBox" name="drawableGroupBox">
         <property name="title">
          <string>Item Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="itemNameLabel">
              <property name="text">
               <string>(Item Name)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="hideBtn">
              <property name="text">
               <string>&amp;Hide</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="itemPropBase" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QFormLayout" name="itemPropLayout"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="playlistSetupWidget" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0" rowspan="5">
              <widget class="QListView" name="playlistView"/>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="addToPlaylistBtn">
               <property name="text">
                <string>Add to Play&amp;list</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="sendToPlayerBtn">
               <property name="text">
                <string>Sen&amp;d to Player</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Length:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="itemLengthBox">
               <property name="suffix">
                <string> sec</string>
               </property>
               <property name="maximum">
                <double>99999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>91</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="timeLabel">
               <property name="text">
                <string>(Playlist Timer)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="pauseBtn">
               <property name="text">
                <string>&amp;Pause</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QPushButton" name="playBtn">
               <property name="text">
                <string>Pl&amp;ay</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSlider" name="fadeSpeedSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fadeBlackBtn">
        <property name="text">
         <string>Black</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_Collection"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Canvas_Size"/>
    <addaction name="separator"/>
    <addaction name="actionPlayer_Setup"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEdit_Selected_Group"/>
    <addaction name="actionAdd_New_Group"/>
    <addaction name="actionRemove_Selected_Group"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Collection">
   <property name="text">
    <string>&amp;Open Collection...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionEdit_Selected_Group">
   <property name="text">
    <string>&amp;Edit Selected Group (Shift+F2)</string>
   </property>
   <property name="shortcut">
    <string>Shift+F2</string>
   </property>
  </action>
  <action name="actionAdd_New_Group">
   <property name="text">
    <string>Add &amp;New Group (CTRL+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionRemove_Selected_Group">
   <property name="text">
    <string>&amp;Delete Selected Group</string>
   </property>
  </action>
  <action name="actionPlayer_Setup">
   <property name="text">
    <string>Players &amp;Connections...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionChange_Canvas_Size">
   <property name="text">
    <string>Change Canvas Size...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditorGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>EditorGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
